docassemble/juvenilesealing/data/questions/juvenile-sealing-text-10.yml

```diff
@@ -109,8 +109,6 @@ subquestion: |
   sheets and those sheets are kept at the courthouse
   that handled that case.
 
-  ## Getting certified copies
```

Comment:
Real quick check in: what's the reasoning behind deleting this and the other format changes to headings in here?

Reply:
Ah, I expanded the previous code. I see the heading was moved up. Have you visually previewed the new headings? Do they style consistently?

Also, it looks like 'Getting certified copies' is at the very top of the page. Maybe it's not worth having it anymore since the title of the page says the same thing.

Thoughts?

Reply:
Y'know what, we can fix that stuff later.

Reply:
> Do they style consistently?

Don't know where my brain is right now. Other headings also are `###` and they style just fine. Proposal to remove top heading stands and it's the only relevant comment in this gosh darned comment thread.

Reply:
I don't understand what top heading refers to here. I've attached a screenshot of what this page currently looks like so we can talk about this tomorrow. 

<img width="740" alt="Screen Shot 2020-02-01 at 11 51 52 PM" src="https://user-images.githubusercontent.com/3387590/73603230-0ca30f80-454e-11ea-93ff-dcafcc38681e.png">